docs/processing/qaqc-criteria-preprocessed.md

```diff
@@ -1,55 +1,51 @@
+  If that is not the case, use the *fMRIPrep* `--bids-filter-file` flag to exclude unwanted T1w from the processing and re-run *fMRIPrep*.
+- [ ] Verify that the standard output space corresponds to the one you want.
+  If that is not the case, assign the correct output space to the `--output-spaces` flag of *fMRIPrep* and re-run *fMRIPrep*.
 
 ## Anatomical
 
 ### Anatomical conformation
 
 ### Brain mask and brain tissue segmentation of the T1w
-
-- [ ] Check for residual intensity non-uniformity (INU).
-  If the INU is clearly visible, tweak the INU correction and re-run *fMRIPrep*.
-- [ ] Check that the brain mask does not cut off part of the brain and/or contain holes surrounding signal drop-out regions.
-  If it is not the case, tweak the brain mask estimation and re-run *fMRIPrep*.
-- [ ] Check that the brain mask does not include parts that are clearly NOT brain.
-  Typically, this appears as bumps surrounding high-intensity areas of signal outside the brain (e.g dura or skull).
-  If it is not the case, tweak skull-stripping and re-run *fMRIPrep*.
 - [ ] Assess the quality of the segmentation:
     - [ ] Check that the blue line correctly outlines the boundary between GM and WM.
     - [ ] Check that the magenta line outlines ventricles.
     - [ ] Check that the contours do not exclude single voxels within piecewise-smooth regions (generally more identifiable in the WM and inside the ventricles).
-      Careful if those single excluded voxels only appear close to tissue boundaries, they are likely misclassified because of partial volume effect, as such it does not consistitute an exclusion criteria.
+      Careful if those single excluded voxels only appear close to tissue boundaries, they are likely misclassified because of partial volume effect, as such it does not constitute an exclusion criteria.
     - [ ] Check that the contours do not include other tissues than the tissue of interest (GM; magenta, WM; blue). 
-    - [ ] If you found any of the issues above, tweak the segmentation and re-run *fMRIPrep*.
+    - [ ] If you found any of the issues above, do not use the CompCor confounds for nuisance regression.
 
 ### Spatial normalization of the anatomical T1w reference
 
 - [ ] In order of importance, the following structures should be correctly aligned: 1. ventricles, 2. subcortical regions, 3. corpus callosum, 4. cerebellum, 5. cortical GM.
     Tweak the spatial normalization and re-run *fMRIPrep* in case of a misalignment of the ventricles, subcortical regions, or the corpus callosum.
     You can, however, be more lenient with GM alignment.
 - [ ] Check for severe stretching or distortion of the T1w.
-  If observed, tweak the normalization and re-run *fMRIPrep*.
-
-### Surface reconstruction
+  If observed, use the preprocessed BOLD images in subject space for the analysis rather than MNI.
```

Comment:
What does this mean?
```suggestion
  If observed, use the preprocessed BOLD images in subject space for the analysis rather than MNI.
```

Reply:
If we run into coregistration issue, we will perform the analysis in subject space